hackathon_site/event/views.py

```diff
@@ -200,3 +205,20 @@ def post(self, request, *args, **kwargs):
         at once.
         """
         return super().post(request, *args, **kwargs)
+
+
+class TeamListView(mixins.ListModelMixin, generics.GenericAPIView):
+    queryset = EventTeam.objects.all()
+    serializer_class = TeamSerializer
+
```

Comment:
Now you need to attach the permissions class you defined:

```
permissions_classes = [FullDjangoModelPermissions]
```

Reply:
@grahamhoyes  Graham, actually I have a question regarding this permission which I forgot to ask you in our last meeting. What is the purpose of this permission? The description in Jira said this "DjangoModelPermissions doesn’t actually check for view permissions on get requests, so do some googling". But I have tried that If I logged out under the case without adding this permission group, I will not be able to use get request. The page will say "not authorized". I cannot see any difference before and after adding this permission group.

Reply:
The DRF `DjangoModelPermissioms` doesn't check any permissions for GET requests, but the one that you've added does.

But now to test it, you'll need to give yourself that permission! On the admin site, edit your user and look for a permission like `Event | Team | View Team` or something like that and add it to yourself. You should be able to view the API after that. 

You'll also need tests for that. When not logged in, you should get a 401 Unauthorized. When logged in without that permission, you should get a 403 Forbidden. Finally with that permission, everything should work.

If you look up Django's permission system, you'll find how you can query and add permissions to a user. I recommend modifying the `_login` test method to take a list of permissions that it will add to the user. Then on your test, you can do something like `self.permisaions = <query for event.event_view permissions>`, and pass those to the login call.

Let me know if you need any help with that!